terraform/modules/locality_kubernetes/kubernetes.tf

```diff
@@ -0,0 +1,141 @@
+variable "environment" {
+  type = string
+}
+
+variable "kubernetes_namespace" {
+  type = string
+}
+
+variable "gcp_project" {
+  type = string
+}
+
+variable "manifest_bucket" {
+  type = string
+}
+
+variable "ingestors" {
+  type = list(string)
+}
+
+# We need to make a google service account for the manifest updater
+resource "google_service_account" "manifest_updater" {
+  provider = google-beta
+
+  account_id   = "prio-${random_string.account_id.result}"
+  display_name = "prio-${var.environment}-${var.kubernetes_namespace}-manifest-updater"
+}
+
+resource "random_string" "account_id" {
+  length  = 16
+  upper   = false
+  number  = false
+  special = false
+}
+
+# This is another kubernetes-level service account which we will associate with the operator GCP
+# service account above.
+resource "kubernetes_service_account" "manifest_updater" {
+  metadata {
+    name      = "manifest-updater"
+    namespace = var.kubernetes_namespace
+    annotations = {
+      environment                      = var.environment
+      "iam.gke.io/gcp-service-account" = google_service_account.manifest_updater.email
+    }
+  }
+}
+
+# This carefully constructed string lets us refer to the Kubernetes service
+# account in GCP-level policies, below. See step 5 in
+# https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#authenticating_to
+locals {
+  manifest_updater_sa = "serviceAccount:${var.gcp_project}.svc.id.goog[${var.kubernetes_namespace}/${kubernetes_service_account.manifest_updater.metadata[0].name}]"
+}
+
+# Bind the GCP and K8s service accounts together
+resource "google_service_account_iam_binding" "manifest_updater_workload" {
+  provider           = google-beta
+  service_account_id = google_service_account.manifest_updater.name
+  role               = "roles/iam.workloadIdentityUser"
+  members = [
+    local.manifest_updater_sa
+  ]
+}
```

Comment:
I don't love having this copy pasted from `kubernetes/kubernetes.tf`. I think maybe you could refactor this into a module that creates a GCP SA, a Kubernetes SA and then brings them together, then use that from both this module and `kubernetes/kubernetes.tf`.